src/AndersonAcceleration.f90

```diff
@@ -296,39 +296,33 @@ SUBROUTINE reset_AndersonAccelerationType(solver,x)
   CLASS(AndersonAccelerationType),INTENT(INOUT) :: solver
   CLASS(VectorType),INTENT(INOUT) :: x
 
-  CHARACTER(LEN=*),PARAMETER :: myName='reset_AndersonAccelerationType'
```

Comment:
Let's leave the parameter in there and use it in the error messages.  That's our typical pattern (especially since it's used in multiple places), so let's stick with it.

Reply:
Can't unfortunately, because the only uses of it are wrapped in ifdefs for tpls that the Futility ci does builds without making it look like that char variable is unused making the ci fail.

Reply:
Just wrap it in ifdefs too... that's what we do elswhere

Reply:
Gross, alright will do.